spec/Section 7 -- Response.md

```diff
@@ -10,71 +10,71 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A GraphQL request returns either a _response_ or a _response stream_.
+:: A GraphQL _response_ is either a _response map_ or a _response stream_.
 
-### Response
+### Response Map
 
-:: A GraphQL request returns a _response_ when the GraphQL operation is a query
-or mutation. A _response_ must be a map.
+:: A GraphQL request returns a _response map_ when the GraphQL operation is a
+query or mutation. A _response map_ must be a map.
 
-If the request raised any errors, the response map must contain an entry with
+If the request raised any errors, the _response map_ must contain an entry with
 key `errors`. The value of this entry is described in the "Errors" section. If
 the request completed without raising any errors, this entry must not be
 present.
 
-If the request included execution, the response map must contain an entry with
+If the request included execution, the _response map_ must contain an entry with
 key `data`. The value of this entry is described in the "Data" section. If the
 request failed before execution, due to a syntax error, missing information, or
 validation error, this entry must not be present.
 
-The response map may also contain an entry with key `extensions`. This entry, if
-set, must have a map as its value. This entry is reserved for implementers to
+The _response map_ may also contain an entry with key `extensions`. This entry,
+if set, must have a map as its value. This entry is reserved for implementers to
 extend the protocol however they see fit, and hence there are no additional
 restrictions on its contents.
 
 To ensure future changes to the protocol do not break existing services and
-clients, the top level response map must not contain any entries other than the
-three described above.
+clients, the top level _response map_ must not contain any entries other than
+the three described above.
 
-Note: When `errors` is present in the response, it may be helpful for it to
-appear first when serialized to make it more clear when errors are present in a
-response during debugging.
+Note: When `errors` is present in the _response map_, it may be helpful for it
+to appear first when serialized to make it more clear when errors are present in
+a _response map_ during debugging.
 
 ### Response Stream
 
 :: A GraphQL request returns a _response stream_ when the GraphQL operation is a
-subscription. A _response stream_ must be a stream of _response_.
+subscription. A _response stream_ must be a stream of _response map_.
```

Comment:
This isn't actually true, a subscription can yield a _response map_ if a request error occurs. This should be reflected in the response map definition too.